test/test_datasets.py

```diff
@@ -29,11 +29,13 @@ def test_librispeech(self):
         data[0]
 
     @unittest.skipIf("sox" not in common_utils.BACKENDS, "sox not available")
+    @common_utils.AudioBackendScope('sox')
```

Comment:
See #715

Reply:
Does this fixes #715 completely?

Reply:
Nope. This is temporary fix. #719 gives principled fix which rely on this PR.